src/Associations.cc

```diff
@@ -23,6 +23,7 @@
  */
 
 #include <cmath>
+#include <math.h>  // for isnan
```

Comment:
I think that if you use `std::isnan()` then `<cmath>` above should already be fine.